action/protocol/staking/handlers.go

```diff
@@ -239,8 +254,9 @@ func (p *Protocol) handleWithdrawStake(ctx context.Context, act *action.Withdraw
 		Address: p.addr.String(),
 		Topics: action.Topics{
 			action.BucketWithdrawAmount,
-			hash.BytesToHash256(byteutil.Uint64ToBytesBigEndian(bucket.Index)),
+			hash.BytesToHash256(p.addr.Bytes()),
 			hash.BytesToHash256(actionCtx.Caller.Bytes()),
+			hash.BytesToHash256(byteutil.Uint64ToBytesBigEndian(bucket.Index)),
```

Comment:
withdraw: from staking protocol address to action caller

Reply:
unify the topics order, so it is the same for all:
topic[1]: from
topic[2]: to